tools/migrate/v1_deprecation.ts

```diff
@@ -0,0 +1,94 @@
+import { ClickHouse } from "@unkey/clickhouse";
+import { mysqlDrizzle, schema } from "@unkey/db";
+import { Resend } from "@unkey/resend";
+import { WorkOS } from "@workos-inc/node";
+import mysql from "mysql2/promise";
+import { z } from "zod";
+async function main() {
+  const ch = new ClickHouse({
+    url: process.env.CLICKHOUSE_URL,
+  });
+
+  const conn = await mysql.createConnection(
+    `mysql://${process.env.DATABASE_USERNAME}:${process.env.DATABASE_PASSWORD}@${process.env.DATABASE_HOST}:3306/unkey?ssl={}`,
+  );
+
+  await conn.ping();
+  const db = mysqlDrizzle(conn, { schema, mode: "default" });
+
+  const workos = new WorkOS(process.env.WORKOS_API_KEY);
+
+  const resend = new Resend({
+    apiKey: process.env.RESEND_API_KEY,
+  });
+  console.log("starting");
+
+  const rows = await ch.querier.query({
+    query: `
+    SELECT
+      workspace_id,
+      splitByChar('?', path, 1)[1] as path
+    FROM metrics.api_requests_per_day_v1
+    WHERE startsWith(path, '/v1/')
```

Comment:
can we exclude people who used the analytics endpoint? since thats not yet available in v2?

Reply:
+1 to this.